Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -301,6 +306,17 @@ void ProcessThrottler::updateThrottleStateIfNeeded()
     updateThrottleStateNow();
 }
 
+#if USE(EXTENSIONKIT_ASSERTIONS)
+void ProcessThrottler::didConnectToProcess(RetainPtr<_SEExtensionProcess> process)
+{
+    PROCESSTHROTTLER_RELEASE_LOG("didConnectToProcess");
```

Comment:
It's too bad `_SEExtensionProces` doesn't give us a way to see the PID of the underlying process. Logging it here would be helpful for debugging.